bundles/mapping/mapwfs2/plugin/WfsVectorLayerPlugin/impl/MvtLayerHandler.ol.js

```diff
@@ -7,6 +7,13 @@ import { FeatureExposingMVTSource } from './MvtLayerHandler/FeatureExposingMVTSo
 import { WFS_ID_KEY, getFieldsAndPropsArrays } from '../util/props';
 import { AbstractLayerHandler } from './AbstractLayerHandler.ol';
 
+const FEATURE_DATA_UPDATE_THROTTLE = 5000;
+const TILEGRID_3067 = {
+    extent: [-548576, 6291456, 1548576, 8388608],
+    resolutions: [8192, 4096, 2048, 1024, 512, 256, 128, 64, 32, 16, 8, 4, 2, 1, 0.5, 0.25],
+    tileSize: [256, 256]
+};
```

Comment:
Still needs to be fixed for other projections but that change is already being implemented. Lets wait until merging this.